XenModel/Messages.resx

```diff
@@ -7161,6 +7161,24 @@ Size: {3}</value>
   <data name="MAC_INVALID" xml:space="preserve">
     <value>Invalid MAC address.</value>
   </data>
+  <data name="MAIL_LANGUAGE_CHINESE_CODE" xml:space="preserve">
+    <value>zh-CN</value>
+  </data>
+  <data name="MAIL_LANGUAGE_CHINESE_NAME" xml:space="preserve">
+    <value>Chinese(S)</value>
+  </data>
+  <data name="MAIL_LANGUAGE_ENGLISH_CODE" xml:space="preserve">
+    <value>en-US</value>
+  </data>
+  <data name="MAIL_LANGUAGE_ENGLISH_NAME" xml:space="preserve">

```

Comment:
We should add a space character before the brackers, e.g. _English (Unites States)_.
Also, for Chinese, wouldn't be better to say _Chinese (Simplified)_ instead of _Chinese (S)_?